src/test/java/jenkins/plugins/git/MergeWithGitSCMExtensionTest.java

```diff
@@ -0,0 +1,95 @@
+package jenkins.plugins.git;
+
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.plugins.git.*;
+import hudson.plugins.git.extensions.GitSCMExtension;
+import hudson.plugins.git.extensions.GitSCMExtensionTest;
+import hudson.plugins.git.extensions.impl.LocalBranch;
+import hudson.plugins.git.extensions.impl.PreBuildMerge;
+import hudson.plugins.git.util.BuildData;
+import junit.framework.TestCase;
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.eclipse.jgit.lib.Constants;
+import org.jenkinsci.plugins.gitclient.MergeCommand;
+import org.junit.Test;
+
+import java.io.*;
```

Comment:
The [CONTRIBUTING guide](https://github.com/jenkinsci/git-plugin/blob/master/CONTRIBUTING.adoc#contributing-to-the-git-plugin) states that new code should follow the [SCM plugin coding style](https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md#code-style-guidelines).

Those guidelines say that [wildcard imports are banned](https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md#imports).  If you'd be willing to replace this wildcard import with regular imports, that would help reduce the style differences between this and other new code being added to the plugin.

Reply:
Yes. I forgot to check coding style. Now,  I optimized and removed the banned import style.

Reply:
Thanks so much! That is kind of you to make the update when the existing code is painfully inconsistent in its formatting.